fakturogen-comarch-connector/src/main/java/pl/fakturogen/comarch/connector/model/ApiInvoice.java

```diff
@@ -0,0 +1,229 @@
+package pl.fakturogen.comarch.connector.model;
+
+import com.fasterxml.jackson.annotation.*;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+        "$id",
+        "PaymentStatus",
+        "PurchasingPartyId",
+        "ReceivingPartyId",
+        "PaymentTypeId",
+        "BankAccountId",
+        "SalesDate",
+        "InvoiceType",
+        "Items",
+        "Description",
+        "IssueDate",
+        "Number",
+        "Status",
+        "Id"
+})
+public class ApiInvoice {
```

Comment:
I think we are using ComarchInvoice name since this json is coming from Comarch api